LethalAPI.Core/Patches/Events/Player/UsingItemPrefix.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="UsingItemPrefix.cs" company="LethalAPI Modding Community">
+// Copyright (c) LethalAPI Modding Community. All rights reserved.
+// Licensed under the GPL-3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+// ReSharper disable InconsistentNaming
+namespace LethalAPI.Core.Patches.Events.Player;
+
+using LethalAPI.Core.Events.Attributes;
+using LethalAPI.Core.Events.EventArgs.Player;
+
+/// <summary>
+///     Patches the <see cref="HandlersPlayer.UsingItem"/> event.
+/// </summary>
+[EventPatch(typeof(HandlersPlayer), nameof(HandlersPlayer.UsingKey))]
+[HarmonyPatch(typeof(GrabbableObject), nameof(GrabbableObject.ItemActivate))]
+internal static class UsingItemPrefix
+{
+    [HarmonyPrefix]
+    private static bool Prefix(GrabbableObject __instance, bool used, bool buttonDown = true)
```

Comment:
This event call is a prefix, the others are postfixes. They should probably all be postfixes to ensure predictable behaviour?

Reply:
The others are not deniable. This one should be deniable. I think this will have to be reworked to apply in a dynamic approach, as this only affects the base.

Reply:
But this does have to be a prefix to do so easily. The postfixes execute at the end, so the base code won't be interrupted. I think if it's not deniable, it should be a postfix. (if possible - sometimes they require transpilers.)